packages/graphql_flutter/changelog.json

```diff
@@ -1,6 +1,6 @@
 {
   "package_name": "graphql_flutter",
-  "version": "v5.0.2-beta.7",
```

Comment:
Should this be part of this PR?

Reply:
Mh! I think yes, the update file is in the other PR :) I just split the the changes in two